### ISSUE TITLE
Feature Removal

### ISSUE THREAD

**Issue by mistervee** [mistervee/skin.madnox#17]
**Tool tip for the media menu and top bar -  REMOVED**

**Skin Settings - Use quicknav instead of default shutdown menu REMOVED** - Dangerous setting - If for some reason skin shortcuts is removed or bugged there is no way to get to the settings section to change the skin

**Skin Settings - Play trailers in full screen REMOVED** - possible chance of conflicts as you cant have video in the background and a small video window at the same time.

**New view selector** - TO BE REMOVED - Covers the screen and navigation a pain. http://imgur.com/a/Om4Qb - MAD, I'll work on this and do initial update to pull in new views etc.

**First run Help - Auto config** - Not needed if skin settings are coded a certain way.
``` <include condition="!Skin.HasSetting(firstrun4.0)">AutoConfig</include>```

**Overlay for new database items** - massive code bloat too many property's and conditions for the variable.
- Overlay removal for lists? change to overlay on poster?

**Alphabet Jump** - Is this needed? It seems to not fit in with the views and a bit of a novelty. While the standard scrollbar seems to do a fine job.

**Comment by mistervee** [comment]
@bartolomesoriano @meowmoo @MadMikeDoc 
For the list style view we have an overlay for watched, progress ect.

I am thinking that this should be removed and only be visible in the top right corner of the artwork. With the new panel textures there is less space to accommodate this. All other views have like showcase and shift have the overlay texture's in the top right, for consistency i would like to change this. 

So i am proposing using these style textures for watched, progress ect;
https://github.com/MadMikeDoc/skin.aeon.madnox/blob/master/media/overlays/showcase/OverlayWatched.png

and removing this for list style views;
https://github.com/mistervee/skin.madnox/blob/master/media/overlays/list/OverlayWatched.png

**Comment by mistervee** [comment]
@bartolomesoriano @meowmoo @MadMikeDoc 

No.2

Page and Item count in the top bar or any where in the view (for view types only)

Is this required?

The top bar does seem like the place to have this, is this something that is needed? Do we really need to know how many pages and items there are in your library (this can be shown on the home screen in the infobar)?

My reason for asking this is i am trying to reduce clutter in the interface.

**Comment by bartolomesoriano** [comment]
@mistervee, I am OK with the overlays you propose.  As for the item and page counts, for me they are very useful to tell me where I am, since my collection is really big, with over 4500 movies, over 400 tv shows and more than 15,000 albums.  That is something that I would keep or make an option for the user to choose to display or not.

@meowmoo @MadMikeDoc

**Comment by mistervee** [comment]
Fair point for Item Count in the top bar. There is already an option to toggle this off.
